templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,40 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List</h1>
+  <a class="btn btn-primary" href="{% url 'taxi:manufacturer-create' %}">Add manufacturer</a>
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
 
-    {% if manufacturer_list %}
-      <table class="table">
+      {% for manufacturer in manufacturer_list %}
         <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
+          <td>
+            {{ manufacturer.id }}
+          </td>
+          <td>
+            {{ manufacturer.name }}
+          </td>
+          <td>
+            {{ manufacturer.country }}
+          </td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-update' manufacturer.id %}">Update</a>
+          </td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-delete' manufacturer.id %}">Delete</a>
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.